packages/neon/neon/lib/src/blocs/accounts.dart

```diff
@@ -37,19 +37,28 @@ class AccountsBloc extends Bloc implements AccountsBlocEvents, AccountsBlocState
     });
 
     final as = accounts.value;
+
     if (_globalOptions.rememberLastUsedAccount.value && _storage.containsKey(_keyLastUsedAccount)) {
       final lastUsedAccountID = _storage.getString(_keyLastUsedAccount);
-      activeAccount.add(lastUsedAccountID != null ? as.find(lastUsedAccountID) : null);
```

Comment:
For some reason this code failed because the key was there, but the value was null.

Reply:
I assume I was wiped here when I clicked on the notification: https://github.com/provokateurin/nextcloud-neon/blob/35eb2d65799f3e6a5f53b62da8cac868d3e52f86/packages/neon/neon/lib/src/app.dart#L143
But I don't understand how the account could be null.